pgjdbc/src/test/java/org/postgresql/test/jdbc42/SetObject310Test.java

```diff
@@ -300,6 +313,23 @@ private List<String> getZoneIdsToTest() {
     return zoneIdsToTest;
   }
 
+  private void instantTimestamps(Instant instant) throws SQLException {
+    String sql = "select ?::timestamp with time zone, ?::timestamp with time zone";
+    try (PreparedStatement ps = con.prepareStatement(sql)) {
+      ps.setObject(1, instant);
+      ps.setObject(2, instant, Types.TIMESTAMP_WITH_TIMEZONE);
+      try (ResultSet rs = ps.executeQuery()) {
+        rs.next();
+        String noType = rs.getString(1);
```

Comment:
noType really means without timezone, perhaps noType isn't the best name.

Reply:
> noType really means without timezone, perhaps noType isn't the best name.

I guess we call it `noType` just because it's read by `getString` ?

Reply:
because I think the test case is designed to test the `setObject` with/without sql type. I will call it `typedX`/`untypedX`.